tools/cuttlefish-host-image-installer/preseed/after_install_1.sh

```diff
@@ -52,3 +61,37 @@ apt-get install -y network-manager
 
 # Network-manager workaround
 rm -f '/etc/NetworkManager/system-connections/Wired connection 1'
+
+# Install Docker container
+# Add Docker's official GPG key:
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y -q ca-certificates curl
+install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/debian/gpg -o /etc/apt/keyrings/docker.asc
+chmod a+r /etc/apt/keyrings/docker.asc
+
+# Add the repository to Apt sources:
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/debian \
+  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+  tee /etc/apt/sources.list.d/docker.list > /dev/null
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y -q docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+usermod -aG docker vsoc-01
+
+# Inastall nvidia-container-toolkit
```

Comment:
Could we add a conditional check to ensure it only runs if an NVIDIA GPU is present? This will avoid unnecessary installations on systems without NVIDIA GPUs

Reply:
Originally, I had a conditional check. It works locally, but when it runs in preseed, it doesn't work. I am still debugging. This is the code:
nvidia_gpu=$(lspci | grep -i nvidia)
amd_gpu=$(lspci | grep VGA | grep AMD)
if [[ "$nvidia_gpu" != "" ]] && [[ "$amd_gpu" == "" ]]
then
  DEBIAN_FRONTEND=noninteractive apt-get install -y -q --force-yes nvidia-open-kernel-dkms
  DEBIAN_FRONTEND=noninteractive apt-get install -y -q --force-yes nvidia-driver
  DEBIAN_FRONTEND=noninteractive apt-get install -y -q --force-yes firmware-misc-nonfree
fi

I am still working and debugging this issue.

Reply:
I have solution  now. I will have a fix commit soon later today.

Reply:
Added conditional check.